project_stock/models/stock_picking_type.py

```diff
@@ -0,0 +1,9 @@
+# Copyright 2022 Tecnativa - Víctor Martínez
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl)
+from odoo import fields, models
+
+
+class StockPickingType(models.Model):
+    _inherit = "stock.picking.type"
+
+    code = fields.Selection(selection_add=[("ps_operation", "Project stock")])
```

Comment:
Why using a special code for these picking types? Is adding any value?

Reply:
It was added only to have the moves differentiated, but with the `pickign_type_id` field I think it is enough. It is removed.